modules/nw-aws-load-balancer-with-outposts.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+// * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-aws-load-balancer-and-outposts_{context}"]
+= AWS Load Balancer Operator and AWS Outposts
+
+AWS Load Balancer Operator can work with AWS Outposts via the cluster ingress operator. This service needs to be annotated with a non outposts subnet.
+
+Ingress resources also must be annotated with the either regular subnets or outposts subnets (but not both).
```

Comment:
Also, we need to add that NLBs are not supported on AWS Outposts, therefore ALBO will fail to provision them on AWS Outposts subnets.

Reply:
Thank you!